.github/workflows/dotnet-upgrade.yml

```diff
@@ -0,0 +1,167 @@
+name: "DotNet SDK Upgrade"
```

Comment:
make sure you test the workflow in a fork repo and it can create the PR properly.

Reply:
Just forced a re-run on it updating to latest, been testing this all out on a cloned repo [here](https://github.com/bbq-beets/vanzeben-sandbox/actions/runs/5925891927)